src/components/SEP/MeetingComponents/SEPMeetingComponentsView.tsx

```diff
@@ -28,43 +30,57 @@ const SEPMeetingComponentsView: React.FC<SEPMeetingComponentsViewProps> = ({
   const classes = useStyles();
   const downloadSEPXLSX = useDownloadXLSXSEP();
   const { loadingCalls, calls } = useCallsData();
-  // NOTE: Default call is with id=1
-  const [selectedCallId, setSelectedCallId] = useState<number>(1);
+  const [query, setQuery] = useQueryParams({
+    call: withDefault(NumberParam, null),
+  });
+
+  useEffect(() => {
+    if (calls.length && !query.call) {
+      setQuery({ call: calls[0].id });
+    }
+  }, [calls, query.call, setQuery]);
+
+  const getSelectedCall = () => {
+    return calls.find((call) => call.id === query.call);
+  };
 
   const Toolbar = (data: Options<JSX.Element>): JSX.Element => (
     <>
       <MTableToolbar {...data} />
-      <CallFilter
-        calls={calls}
-        isLoading={loadingCalls}
-        onChange={setSelectedCallId}
-        callId={selectedCallId}
-      />
-      <Tooltip title="Export in Excel">
-        <IconButton
-          aria-label="export in excel"
-          className={classes.spacing}
-          data-cy="download-sep-xlsx"
-          onClick={() =>
-            downloadSEPXLSX(
-              sepId,
-              selectedCallId,
-              calls.find(({ id }) => id === selectedCallId)?.shortCode ??
-                'unknown'
-            )
-          }
-        >
-          <GridOnIcon />
-        </IconButton>
-      </Tooltip>
+      <Stack alignItems="center" direction="row">
+        <CallFilter
+          calls={calls}
+          isLoading={loadingCalls}
+          callId={query.call}
+        />
+        <Tooltip title="Export in Excel">
+          <IconButton
+            aria-label="export in excel"
+            className={classes.spacing}
+            data-cy="download-sep-xlsx"
+            onClick={() => {
+              if (query.call) {
```

Comment:
Maybe you should disable button instead. If it is clickable and nothing happens that could be confusing

Reply:
I see now that the table is disabled anyway when there is no call. So you can't click the button while the table is loading. And also there is no way to remove selection so you must have some call selected.